js/piwik.js

```diff
@@ -7281,9 +7281,9 @@ if (typeof window.Piwik !== 'object') {
              */
             addTracker: function (piwikUrl, siteId) {
                 if (!asyncTrackers.length) {
-                    createFirstTracker(piwikUrl, siteId);
+                    return createFirstTracker(piwikUrl, siteId);
                 } else {
-                    asyncTrackers[0].addTracker(piwikUrl, siteId);
+                    return asyncTrackers[0].addTracker(piwikUrl, siteId);
                 }
```

Comment:
JSLint fails here. Guess it needs to be
```javascript
var tracker;
if (!asyncTrackers.length) {
    tracker = createFirstTracker(piwikUrl, siteId);
} else {
    tracker = asyncTrackers[0].addTracker(piwikUrl, siteId);
}
return tracker;
```

Reply:
thx @sgiehl fixed it, JSLint tests pass now